docs/workforce-management-framework/topics/con-release-notes.adoc

```diff
@@ -0,0 +1,25 @@
+= {PROJECT_NAME} {WFM-RC-Version} Release Notes
```

Comment:
WDYT about some lightweight title like `What's new` instead of `Release Notes`?

@austincunningham @paolobueno ?

Reply:
@wtrocki  alot of other products use `Release Notes` , think we should stick with convention.